src/components/AnchorForAttachmentsOnly/BaseAnchorForAttachmentsOnly.tsx

```diff
@@ -48,7 +49,7 @@ function BaseAnchorForAttachmentsOnly({style, source = '', displayName = '', dow
                             return;
                         }
                         Download.setDownload(sourceID, true);
-                        fileDownload(sourceURLWithAuth, displayName).then(() => Download.setDownload(sourceID, false));
+                        fileDownload(sourceURLWithAuth, displayName, '', Browser.isMobileSafari()).then(() => Download.setDownload(sourceID, false));
```

Comment:
Why limit this to `mobileSafari` only? It was previously working on web and native.

Reply:
@allroundexperts Do you think this is clear based on [this](https://github.com/Expensify/App/pull/37131#discussion_r1525410433)?